### PR TITLE
💄 Focus ring visibility

### LINKED ISSUE #1733: Make focus-indicators more uniform across components
Both clickbounds and focus-indicators vary greatly across components, as can be seen in the screenshot. Most components have dotted outlines, but not all – and the spacing between the components and the focus-indicators are inconsistent. 

image

### PR DESCRIPTION
**Pull request by oddvernes**
resolves #1733

### CODE REVIEW

File: packages/eds-core-react/src/components/Button/tokens/icon.ts [hunk 1]
```diff
@@ -61,7 +64,7 @@ export const primary: ButtonToken = mergeDeepRight(button, {
     },
     focus: {
       outline: {
-        offset: '0px',
+        offset: `-${parseInt(focusOutlineWidth)}px`,
```
Comment (mimarz) [inline packages/eds-core-react/src/components/Button/tokens/icon.ts:67]: This was the fix for having the focus frame "fit" when used inside table?

Comment (oddvernes) [reply packages/eds-core-react/src/components/Button/tokens/icon.ts:67]: Yes the ghost icon button

Comment (mimarz) [reply packages/eds-core-react/src/components/Button/tokens/icon.ts:67]: Its just weird if you have it in focus and hover over it, but who am I kidding. Like thats ever going to be a common use-case 🤪

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**